MetamodelAlgemeen.md

```diff
@@ -950,11 +952,7 @@ Een informatiemodel moet echter ook gebruikt kunnen worden zonder dat er vastges
   <dfn>Modelelement identificatie</dfn>De identificatie van een modelelement.
```

Comment:
Waarom een spatie in `modelelement identificatie`? In normaal Nederlands zou het `modelelementidentificatie` zijn. We zetten ook geen kunstmatige spaties bij andere metagegevens voor zover ik kan zien.

Reply:
is er ingeslopen denk ik ik zal de spatie ff weghalen.